internal/resource/configmap_test.go

```diff
@@ -226,11 +226,55 @@ CONSOLE_LOG=new`
 ssl_options.certfile  = /etc/rabbitmq-tls/tls.crt
 ssl_options.keyfile   = /etc/rabbitmq-tls/tls.key
 listeners.ssl.default = 5671
+
+management.ssl.certfile   = /etc/rabbitmq-tls/tls.crt
+management.ssl.keyfile    = /etc/rabbitmq-tls/tls.key
+management.ssl.port       = 15671
 `)
 
 				Expect(configMapBuilder.Update(configMap)).To(Succeed())
 				Expect(configMap.Data).To(HaveKeyWithValue("rabbitmq.conf", expectedRabbitmqConf))
 			})
+
+			When("additional plugins are enabled", func() {
+				It("adds TLS config for the additional plugins", func() {
+					instance = rabbitmqv1beta1.RabbitmqCluster{
+						ObjectMeta: metav1.ObjectMeta{
+							Name: "rabbit-tls",
+						},
+						Spec: rabbitmqv1beta1.RabbitmqClusterSpec{
+							TLS: rabbitmqv1beta1.TLSSpec{
+								SecretName: "tls-secret",
+							},
+							Rabbitmq: rabbitmqv1beta1.RabbitmqClusterConfigurationSpec{
+								AdditionalPlugins: []rabbitmqv1beta1.Plugin{
+									"rabbitmq_mqtt",
+									"rabbitmq_stomp",
+									"rabbitmq_amqp_1_0",
+								},
+							},
+						},
+					}
```

Comment:
Neat pick: perhaps we can use `generateRabbitmqCluster()` and modify only the bits an pieces we want to test?

https://github.com/rabbitmq/cluster-operator/blob/911ef0e91ce9028599ee6539529af09397613ae9/internal/resource/statefulset_test.go#L1572-L1579

Something in the lines of:

```golang
instance = generateRabbitmqCluster()
instance.spec.ObjectMeta.Name = "rabbit-tls"
instance.spec.TLS.SecretName = "a-secret"
instance.spec.Rabbitmq.AdditionalPlugins = []rabbitmqv1beta1.Plugin{ ... }
```

Reply:
That's a good idea!

Reply:
I like this idea too. In the interest of time I will create another issue to refactor the config map tests. Thanks Aitor